source/com.android.installreferrer/installreferrer/Transforms/Metadata.Namespaces.xml

```diff
@@ -0,0 +1,27 @@
+<metadata>
+    <attr
+        path="/api/package[@name='com.android.installreferrer']"
+        name="managedName"
+        >
+        Xamarin.Android.InstallReferrer
+    </attr>
+    <attr
+        path="/api/package[@name='com.android.installreferrer.api']"
+        name="managedName"
+        >
+        Xamarin.Android.InstallReferrer.API
```

Comment:
It appears that GitHub ate my other comment...

This should be `Api`:

> The PascalCasing convention, used for all identifiers except parameter names, capitalizes the first character of each word (including acronyms over two letters in length)

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers

Reply:
fixed